content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -152,6 +152,17 @@ Pick a control plane node that you wish to upgrade first. It must have the `/etc
    [upgrade/kubelet] Now that your control plane is upgraded, please proceed with upgrading your kubelets if you haven't already done so.
    ```
 
+   {{< note >}}
+   Before v1.27 (including v1.27), kubeadm will upgrade the addons (including CoreDNS and kube-proxy) immediately during 
+   `kubeadm upgrade apply`, regardless of whether there are other control plane instances that have not been upgraded, which 
+   may cause compatibility problems. Since v1.28, kubeadm will always checks whether all the control plane instances have 
```

Comment:
```suggestion
   may cause compatibility problems. Since v1.28, kubeadm defaults to a mode that checks whether
  all the control plane instances have
```